_includes/statsDashboard.html

```diff
@@ -0,0 +1,101 @@
+<h2>{{ site.data.i18n.general.stats.projects[page.lang] }}</h2>
+<div id="chartContainer_codebases">
+  <!-- Data Table -->
+  <div>
+    {%- for govLevels in site.data.code -%}
+      {%- assign govLevel = govLevels[0] -%}
+
+      {%- if govLevels[1].size > 0 -%}
+      <h3>{{ site.data.i18n.administrations[govLevel][page.lang] }}</h3>
+      <table class="wb-tables table table-striped table-hover wb-charts wb-charts-bar" id="dataset-filter-{{govLevels[0]}}" data-wb-tables='{"order": [1, "asc"], "columnDefs": [{"targets": [0,5,6,8,9,10,11,12], "visible": false}], "paging": true}'>
```

Comment:
Adding the `caption` value to the table will let it auto-collapse to a details summary